packages/client/src/client/OpenFin/Window/WindowFrame.styles.tsx

```diff
@@ -16,51 +16,63 @@ export const FrameRoot = styled.div`
   width: 100%;
   height: 100%;
   overflow: hidden;
-  background-color: ${({ theme }) => theme.core.darkBackground};
-  color: ${({ theme }) => theme.core.textColor};
+  background-color: ${({ theme }) =>
+    theme.newTheme.color["Colors/Background/bg-primary"]};
+  color: ${({ theme }) =>
+    theme.newTheme.color["Colors/Text/text-primary (900)"]};
 
   --header-height: 3.5rem;
   --footer-height: 2.5rem;
-  --color-behind-views: ${({ theme }) => theme.core.darkBackground};
+  --color-behind-views: ${({ theme }) =>
+    theme.newTheme.color["Colors/Background/bg-primary"]};
 
   #layout-container {
     height: 100%;
     width: 100%;
     padding: 0;
-    background-color: ${({ theme }) => theme.core.darkBackground};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-primary"]};
   }
 
   .wrapper_title {
-    color: ${({ theme }) => theme.core.textColor};
+    color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Text/text-primary (900)"]};
   }
 
   .lm_tab {
-    background-color: ${({ theme }) => theme.core.lightBackground};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-secondary_subtle"]};
     font-size: 3rem;
   }
 
   .lm_tabs {
-    background-color: ${({ theme }) => theme.core.darkBackground};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-primary"]};
     box-sizing: border-box !important;
     padding-left: 1rem;
   }
 
   .lm_header {
-    background-color: ${({ theme }) => theme.core.darkBackground};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-primary"]};
   }
 
   .lm_goldenlayout {
-    background-color: ${({ theme }) => theme.core.lightBackground};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-secondary_subtle"]};
   }
 
   .lm_tab,
   .lm_tab.lm_active {
-    background-color: ${({ theme }) => theme.core.darkBackground};
-    color: ${({ theme }) => theme.core.textColor};
+    background-color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Background/bg-primary"]};
+    color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Text/text-primary (900)"]};
   }
 
   .lm_title {
-    color: ${({ theme }) => theme.core.textColor} !important;
+    color: ${({ theme }) =>
+      theme.newTheme.color["Colors/Text/text-primary (900)"]} !important;
```

Comment:
later, let's see if we can improved the specificity or something, and remove the `!important`